tests/hypertrace/agent/config/test_agent-config.yaml

```diff
@@ -0,0 +1,27 @@
+service_name: pythonagent_001
+reporting:
+  endpoint: http://localhost:9411/api/v2/spans
+  secure: true
+  trace_reporter_type: ZIPKIN
+  token: "TestToken"
+  opa:
+    poll_period_seconds: 50
+    enabled: true
+data_capture:
+  http_headers:
+    request: false
+    response: false
+  http_body:
+    request: false
+    response: false
+  rpc_metadata:
+    request: false
+    response: false
+  rpc_body:
+    request: false
+    response: false
+  body_max_size_bytes: 123457
+propagation_formats: ["B3"]
```

Comment:
can we use `TRACECONTEXT`

Reply:
Same reason as above for OTLP comment, this defaults to tracecontext, we use b3 here to ensure that file config overrides default values.